Samples/MapScenarios/MapScenarios/Extensions/PWRouteInstruction+Helper.swift

```diff
@@ -14,18 +14,23 @@ extension PWRouteInstruction {
         return .systemFont(ofSize: 15.0, weight: .bold)
     }
     
+    func instructionStringForUser() -> String {
+        return attributedInstructionString(highlightedTextColor: .black, regularTextColor: .black).string
+    }
+    
     func attributedInstructionString(highlightedTextColor: UIColor, regularTextColor: UIColor) -> NSAttributedString {
```

Comment:
Should this be private or is it used somewhere else? If it is only used by the above function it should probably be hidden.

Reply:
Used in RouteInstructionCell and TurnByTurnInstructionCollectionViewCell as well

Reply:
I was talking about line 21. I know the line 17 function would be called by external objects but I didn't see any references to "attributedInstructionString" outside of this extension.

Reply:
yeah i was talking about line 21 as well, it's used by those cells
![image](https://user-images.githubusercontent.com/20174191/53653108-1de1ee80-3c10-11e9-91e9-10f5b2d2a257.png)

Reply:
Ok, got it. Stand corrected.